src/WebApi/Infrastructure/Repositories/InterviewQuestionRepository.cs

```diff
@@ -39,5 +39,20 @@ public async Task<IEnumerable<QuestionModel>> Get(GetInterviewQuestionInput inpu
         var result = await interviewQuestions.ToListAsync();
 
         return _mapper.Map<IEnumerable<QuestionModel>>(result);
+    }
+
+    public Task<IEnumerable<QuestionModel>> GetQuestionsBySetId(int id)
```

Comment:
Any reason why this method is not async?

Reply:
there is nothing async in it so it doesn't really make sense.

Reply:
I will change implementation to use FirstOrDefaultAsync instead and make it async 👍